test/models/localization_files_test.rb

```diff
@@ -142,19 +142,21 @@ def source_files(*paths, &block)
     end
   end
 
+  # FIXME: This test needs to search through files
```

Comment:
Should this comment go away?  Looks like you've done it.

Reply:
Thanks. Removed FIXME comment